manual/manual/tutorials/coreexamples.etex

```diff
@@ -596,8 +596,8 @@ expression, "2". Let us see how we initialize a lazy expression.
 We added "print_endline \"lazy_two evaluation\"" to see when the lazy
  expression is being evaluated.
 
-The value of "lazy_two" is "<lazy>", which means the expression is not evaluated 
-yet, and the result is unknown.
+The value of "lazy_two" is displayed as "<lazy>", which means the expression 
+has not been evaluated yet, and its actual value is unknown.
```

Comment:
Final nitpick: maybe more `final value` than `actual value`?

Reply:
Yes, that makes sense because "actual value" might also pertain to value displayed as <lazy>.